<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>885</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>873</width>
    <height>885</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>873</width>
    <height>885</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Panel</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/main_path/images/crio_panel.png);</string>
  </property>
  <widget class="QLCDNumber" name="lcd_p1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_g2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_g1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_g3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_p2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>650</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string>G1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>510</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string>G2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>390</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string>G3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string>P1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>717</x>
     <y>640</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string>P2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_2">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_4">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_5">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_6">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>487</x>
     <y>130</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_7">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>430</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_8">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>460</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_9">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>475</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_10">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>475</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_11_A">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>530</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_11_B">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>720</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_12_A">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>650</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_12_B">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>794</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_13_A">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>650</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_13_B">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>794</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_1_A">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_2_A">
   <property name="geometry">
    <rect>
     <x>716</x>
     <y>510</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_4_A">
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>577</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_5_A">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>577</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pump_4he">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="pump_3he">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="pump_roots">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>130</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="pump_noname">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_14">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>430</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_closed.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_3_A">
   <property name="geometry">
    <rect>
     <x>593</x>
     <y>190</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve_open.png);</string>
   </property>
   <property name="text">
    <string>
O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valve_nd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>675</x>
     <y>280</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>padmaa</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main_path/images/valve.png);</string>
   </property>
   <property name="text">
    <string>
C</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource_file.qrc"/>
 </resources>
 <connections/>
</ui>
